FamilyTree/po/it-local.po

```diff
@@ -19,291 +19,292 @@ msgstr ""
 #: FamilyTree/FamilyTree.py:675
 #, python-format
 msgid "%(father)s and %(mother)s"
-msgstr ""
+msgstr "%(father)s e %(mother)s"
 
 #: FamilyTree/FamilyTree.py:682
 msgid "Anonymous"
-msgstr ""
+msgstr "Anonimo"
 
 #: FamilyTree/FamilyTree.py:905
 msgid "born"
-msgstr ""
+msgstr "nato"
 
 #: FamilyTree/FamilyTree.py:907
 msgid "baptised"
-msgstr ""
+msgstr "battezzato"
 
 #: FamilyTree/FamilyTree.py:909
 msgid "died"
-msgstr ""
+msgstr "deceduto"
 
 #: FamilyTree/FamilyTree.py:911
 msgid "buried"
-msgstr ""
+msgstr "seppellito"
 
 #: FamilyTree/FamilyTree.py:913
 msgid "cremated"
-msgstr ""
+msgstr "cremato"
 
 #: FamilyTree/FamilyTree.py:915
 msgid "married"
-msgstr ""
+msgstr "sposato"
 
 #: FamilyTree/FamilyTree.py:917
 msgid "divorced"
-msgstr ""
+msgstr "divorziato"
 
 #: FamilyTree/FamilyTree.py:919
 msgid "resident"
-msgstr ""
+msgstr "residente"
 
 #: FamilyTree/FamilyTree.py:926
 #, python-format
 msgid "on %(ymd_date)s"
-msgstr ""
+msgstr "il %(ymd_date)s"
 
 #: FamilyTree/FamilyTree.py:928
 #, python-format
 msgid "in %(ym_date)s"
-msgstr ""
+msgstr "il %(ym_date)s"
 
 #: FamilyTree/FamilyTree.py:930
 #, python-format
 msgid "in %(y_date)s"
-msgstr ""
+msgstr "nel %(y_date)s"
 
 #: FamilyTree/FamilyTree.py:934
 #, python-format
 msgid "on %(placeholder)s"
-msgstr ""
+msgstr "in %(placeholder)s"
 
 #: FamilyTree/FamilyTree.py:937
 #, python-format
 msgid "on %(placeholder)s (%(partial)s)"
-msgstr ""
+msgstr "in %(placeholder)s (%(partial)s)"
 
 #: FamilyTree/FamilyTree.py:954
 #, python-format
 msgid "in %(place)s"
-msgstr ""
+msgstr "in %(place)s"
 
 #: FamilyTree/FamilyTree.py:965
 #, python-format
 msgid "(%(description)s)"
-msgstr ""
+msgstr "(%(description)s)"
 
 #: FamilyTree/FamilyTree.py:1043
 msgid "Tree Options"
-msgstr ""
+msgstr "Opzioni per l'albero"
 
 #: FamilyTree/FamilyTree.py:1045
 msgid "Center Family"
-msgstr ""
+msgstr "Famiglia principale"
 
 #: FamilyTree/FamilyTree.py:1046
 msgid "The center family for the tree"
-msgstr ""
+msgstr "La famiglia principale dell'albero"
 
 #: FamilyTree/FamilyTree.py:1049
 msgid "Ancestor Generations"
-msgstr ""
+msgstr "Generazioni ascendenti"
 
 #: FamilyTree/FamilyTree.py:1050
 msgid "The number of ancestor generations to include in the tree"
-msgstr ""
+msgstr "Il numero di generazioni di antenati da includere nell'albero"
 
 #: FamilyTree/FamilyTree.py:1053
 msgid "Descendant Generations"
-msgstr ""
+msgstr "Generazioni discendenti"
 
 #: FamilyTree/FamilyTree.py:1054
 msgid "The number of descendant generations to include in the tree"
-msgstr ""
+msgstr "Il numero di generazioni di discendenti da includere nell'albero"
 
 #: FamilyTree/FamilyTree.py:1057
 msgid "Kekule number of husband"
-msgstr ""
+msgstr "Numero Kekule di mariti"
 
 #: FamilyTree/FamilyTree.py:1058
 msgid ""
 "The Kekule number of the husband (central family). Set 0 to not show Kekule "
 "numbers"
-msgstr ""
+msgstr "Il numero Kekule del marito (famiglia principale). Scegli 0 per non mostrarlo "
+"numbers"
```

Comment:
"numbers " should be removed now, and the space at the end of the previous line (after mostrarlo) should be removed as well.